source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -328,8 +328,10 @@ def wrapper(*args, **kwargs):
                     args['is_journey_schedules'] = True
                     allowed_ids.update(args.get('allowed_id[]', []))
                     args['allowed_id[]'] = list(allowed_ids)
+                    args['_type'] = 'journeys'
+
```

Comment:
extra blank line + why don't you do the same for `rel`? (this stands for the whole PR)

Reply:
because rel is not an optional parameter